### ISSUE TITLE
ov767x library works on ArduinoCore-nrf528x board but not the ArduinoCore-mbed

### ISSUE THREAD

**Issue by hpssjellis** [arduino-libraries/Arduino_OV767X#7]
As far as I can tell this library compiles fine for both the Nano-33-Ble on the [ArduinoCore-nrf528x](https://github.com/arduino/ArduinoCore-nRF528x-mbedos/tree/master/variants/ARDUINO_NANO33BLE) board software and the [ArduinoCore-mbed](https://github.com/arduino/ArduinoCore-mbed/tree/master/variants/ARDUINO_NANO33BLE) board software made for the new Portenta-H7 and Nano-33-Ble, but for the later the image does not show in the processing app. Also the board LED seems to be flashing as if it is trying to flash every time it sends a serial command (which would explain a massive slow down).   

@mbanzi or @sbhklr or others, should this issue be in the [https://github.com/arduino/ArduinoCore-mbed](https://github.com/arduino/ArduinoCore-mbed) github, or should it stay here as an issue for the Arduino_ov767x library?

**Comment by hpssjellis** [comment]
Update on this. I tried using the OV7670 Camera module using the new mbed Core 1.3.0 for the Nano 33 BLE and the PortentaH7. It would not work, however when I downgraded the board to the previous version 1.1.6 it worked fine. Any chance someone can look into this. Perhaps this can also be connected to  [/issues/6](https://github.com/arduino-libraries/Arduino_OV767X/issues/6)   to get the module working on the PortentaH7
